wagtail/admin/templates/wagtailadmin/base.html

```diff
@@ -3,21 +3,20 @@
 
 {% block furniture %}
     {% slim_sidebar_enabled as slim_sidebar_enabled %}
+    <template data-wagtail-sidebar-branding-logo>{% block branding_logo %}{% endblock %}</template>
```

Comment:
This and below – refactoring was needed so we could retain the single `branding_logo` block. The backwards-compatible implementation with the inline script tag for the legacy sidebar is pretty poor, but hopefully good enough for a single release.